### DISCUSSION TITLE
Add more bands for quick band shifting (4m, 12m, 17m, 30m)

### DISCUSSION THREAD

**Discussion by cbspidi** [mbridak/not1mm#117] [Ideas]
**Describe the solution you'd like**

Please add more bands under configuration settings/bands (which will have an effect on selecting the operating band (next to the callsign window).

Please add the below bands:
4m, 
12m
17m 
30m

**Describe alternatives you've considered**
Manual band swap on the radio

**Comment by mbridak** [comment]
Those are WARC bands. No contesting is allowed.

**Comment by cbspidi** [comment]
Yeah, I get your point. 
I use your program (the best I have come across to this point) as a general logger.
With the above, I have QSOs on the mentioned bands in SSB mode, just not in contest mode.

**Comment by mbridak** [comment]
Have you tried QLog?
It's a way better general logger.

**Comment by cbspidi** [comment]
Okay, thanks 

I will have a look(no i have not tried it before), what I like about your take on  login is
its simple
its all in python 
cross-platform 
very portable

**Comment by cbspidi** [comment]
I did have a look. 
Yeah, it's much more advanced and has other functionality, but I still like your program better. :-)

**Comment by cbspidi** [comment]
Hey, @mbridak 
Could you give me a hint on where I would go to change/add this if I were to do it myself?
It's quicker to ask than to reverse engineer :-)

**Comment by mbridak** [comment]
Sure, I looked through the change log to find where I added VHF frequencies, and these appear to be relevant commits to add another band.

https://github.com/mbridak/not1mm/commit/28ee2ca3456b54d2e131bdf2a283fa6d7ed7b68c
https://github.com/mbridak/not1mm/commit/d3944da7156336233a1b8af16e5c836ff08fb701
https://github.com/mbridak/not1mm/commit/b659b851cd68f052a9fb33fe7b2f15b7046e9e31
https://github.com/mbridak/not1mm/commit/a96d8b42340459955ecc8f3317a900e7c8327005
https://github.com/mbridak/not1mm/commit/d3944da7156336233a1b8af16e5c836ff08fb701

Hope that helps.
